src/firebase/events.ts

```diff
@@ -96,3 +98,10 @@ export async function deleteEvent(event: Event) {
     throw new BaseError('Event not found', 404);
   }
 }
+
+export async function userCheckOrgId(userToken: string, orgId: number) {
+  const user: User = await getUserByToken(userToken);
+  if (!user.orgId.includes(orgId)) {
+    throw new BaseError('User is not apart of this org', 401);
```

Comment:
Just to avoid abbreviations in error messages. Otherwise very nice.

```suggestion
    throw new BaseError('User is not apart of this organization', 401);
```